primitives/runtime-interface/src/impls.rs

```diff
@@ -532,3 +532,7 @@ impl PassBy for sp_wasm_interface::ValueType {
 impl PassBy for sp_wasm_interface::Value {
 	type PassBy = Codec<sp_wasm_interface::Value>;
 }
+
+impl PassBy for sp_stats::UsageInfo {
```

Comment:
That is wrong. `sp-stats` should include `runtime-interface` and use the appropriate derive macro.

Reply:
This was what I did in the first place, but then I had some cyclic dependencies.
The problem is `sp-externalities` needs to use `UsageInfo`. But if I add `sp-stats` to `sp-externalities` we get a cyclic dependency (sp-externalities -> sp-stats -> sp-runtime-interface -> sp-externalities).